### DISCUSSION TITLE
Contact Form Submission - Request - Missing Section or Editorial

### DISCUSSION THREAD

**Discussion by maggieliu05** [cpinitiative/usaco-guide#2690] [Ideas]
Someone submitted the contact form!

**Module**: Gold - Range DP
**Topic**: Request - Missing Section or Editorial
**Message**: 
Request solution for BOI 2009 - Beetle

**Comment by bqi343** [comment]
BOI solution sketches can be found here: https://boi.cses.fi/tasks.php

**Comment by bqi343** [comment]
would be resolved by #904
